server/api/story/__tests__/story.spec.js

```diff
@@ -51,5 +51,5 @@ test.serial('Should correctly give count of all stories sorted by name', async t
     .expect('Content-Type', /json/)
   const got = res.body
   t.is(t.context.stories.length, got.length)
-  t.is(got[0].name, stories[0].name)
+  t.is(got.name, stories.name) // TODO: have to sort with datePublished to prove the oldest stories
```

Comment:
This is a good example of a test that cannot fail. 
If got is an array then got.name is undefined.  ditto stories.name so the tests always passes because you compared two undefined values. 

Use node to prove to your self that .value of an array doesn't throw an error and just returns undefined.

Reply:
Hi Andrew. Could you explain what should I do for this comment?

Reply:
I'm a little bit can't understand

Reply:
@avowkind